### PR TITLE
UI api encode URI component for path parameters

### PR DESCRIPTION
**Pull request by nopcoder**
Encode all URI parameters we encode on API calls.
Example: `/auth/users/${userId)`
The `userId` value can be 'my.name+test@gmail.com` which needs to be encoded as 'my.name%2Btest@gmail.com` when used as URL. 
Without encoding, the value will not match while processing on the receiver side.

### CODE REVIEW

**Comment by nopcoder** [comment]
@guy-har ping

File: webui/src/lib/api/index.js [hunk 1]
```diff
@@ -226,57 +226,57 @@ class Auth {
...
-        const response = await apiRequest(`/auth/users/${userId}/policies/${policyId}`, {method: 'DELETE'});
+        const response = await apiRequest(`/auth/users/${encodeURIComponent(userId)}/policies/$encodeURIComponent({policyId)}`, {method: 'DELETE'});
         if (response.status !== 204) {
             throw new Error(await extractError(response));
         }
     }
 
     async attachPolicyToGroup(groupId, policyId) {
-        const response = await apiRequest(`/auth/groups/${groupId}/policies/${policyId}`, {method: 'PUT'});
+        const response = await apiRequest(`/auth/groups/${encodeURIComponent(roupId)}/policies/${encodeURIComponent(policyId)}`, {method: 'PUT'});
```
Comment (guy-har) [inline webui/src/lib/api/index.js:265]:
```suggestion
        const response = await apiRequest(`/auth/groups/${encodeURIComponent(groupId)}/policies/${encodeURIComponent(policyId)}`, {method: 'PUT'});
```

Comment (nopcoder) [reply webui/src/lib/api/index.js:265]: Great catch! thanks.

**Review by guy-har — APPROVED** [review]
LGTM
Thanks

**Pull request merged**